pkg/uroot/util/root.go

```diff
@@ -230,3 +233,31 @@ func Rootfs() {
 	}
 
 }
+
+func GetRSDP() (string, error) {
```

Comment:
This function is much different to the others in this file. Can you move it to a new package named "rsdp"?